docs/learn/local_models.md

```diff
@@ -0,0 +1,34 @@
+---
+search:
+  boost: 3
+---
+
+# Local Models
```

Comment:
I think we should title this "Open-Source Models" instead of local (since they can still be served in the cloud using e.g. vLLM). WDYT?

Reply:
```suggestion
# Open-Source Models
```